src/assets/index.css

```diff
@@ -0,0 +1,131 @@
+body {
+  margin: 0;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  color: #f7dbe6;
+  background: #cc6289;
+  font-size: 15px;
+}
```

Comment:
- [ ] Kindly use the Lato font as required in the project requirement. It's a Google font, you can add/import it to your CSS and use it in the body. Please add that 👍🏽

<img width="1392" alt="Screenshot 2023-08-03 at 6 22 42 PM" src="https://github.com/yemidada/metric-webapp/assets/50941074/5db5f094-bda9-4258-b9b3-a4a1e333534a">